### PR TITLE
#224 Added functionality to update program_sequence

### PR DESCRIPTION
**Pull request by GauravGusain98**
#224 

### Types of changes

- [ ] Bug fix (a change which fixes an issue)
- [x] New feature (a change which adds functionality)

### Short description of what this resolves:

- Update program sequence whenever a call log is created.

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [x] The code changes are passing the CI checks
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: api/services/call_log_service.py
Comment (GauravGusain98) [inline api/services/call_log_service.py:101]: `datetime.now()` gives the system current time. In cloud function the system default timezone is UTC so it was working fine till now but we should use `utcnow()` instead

Comment (GauravGusain98) [reply api/services/call_log_service.py:101]: Also the created_on and updated_on field should be in UTC.

File: api/models/program_sequence.py
Comment (Satendra-SR) [inline api/models/program_sequence.py:28]: Let's confirm on this one before merging

File: api/services/call_log_service.py
Comment (Satendra-SR) [reply api/services/call_log_service.py:101]: Yea, make sense to use UTC explicitly. Though, the default server date-time is in UTC only.

Also, as you are updating it here, just checking if you have checked at other places as well? It will be hard to maintain two different time zone on different machines.

**Review by Satendra-SR — APPROVED** [review]
One question else looks good.

Let's wait for confirmation before merging the PR.

File: api/models/program_sequence.py
Comment (Satendra-SR) [reply api/models/program_sequence.py:28]: Let's remove `sequence_index` and `content_status`

**Pull request merged**
